xarray/conventions.py

```diff
@@ -502,6 +507,20 @@ def stackable(dim):
                     new_vars[k].encoding["coordinates"] = coord_str
                     del var_attrs["coordinates"]
                     coord_names.update(var_coord_names)
+            if "bounds" in var_attrs:
+                bounds_str = var_attrs["bounds"]
+                var_bounds_names = [bounds_str]
+                if all(k in variables for k in var_bounds_names):
```

Comment:
this is always a 1 element list. Can we just say `if bounds_str in variables`?

Reply:
Done.